helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -180,6 +180,8 @@ def _track_template(self, tracking):
             res["stage_id"] = (
                 ticket.stage_id.mail_template_id,
                 {
+                    # Need to set mass_mail so that the email will always be sent
+                    "composition_mode": "mass_mail",
```

Comment:
I'm not sure about this. If you are doing this to have a `mail.mail` record, that's something not needed. `mail.message` is enough.

Reply:
Example of use case:
- An email template is set in the New stage.
- A ticket is created (from the portal form for example).

Before this change:
- Although the chatter indicated the sending of the email, it was not really sent (the recipient is the author).

After this change:
- The email is always sent to the `partner_id` (that of the user who created the ticket).